File: C/Others/Area_of_cylinder.c
#include<stdio.h>
int main(){
    float pi=3.14,radius,height;
   printf("Enter the radius =");
    scanf("%f",&radius);
    printf("Enter the height =");
    scanf("%f",&height);
    printf("Area of cylinder =%.2f",pi*radius*radius*height);
    return 0;
}

File: Java/Area_of_cylinder.java
import java.util.*;
public class Area_of_cylinder {
    public static void main(String[] args) {
        float pi=3.14f;
        Scanner sc= new Scanner(System.in);
        System.out.print("Enter the radius = ");
        float radius= sc.nextFloat();
        System.out.print("Enter the height = ");
        float height= sc.nextFloat();
        
        System.out.print("Area of cylinder ="+pi*radius*radius*height );

        sc.close();

    }
}
